.github/workflows/release.yml

```diff
@@ -32,8 +36,31 @@ jobs:
                   username: ${{ secrets.DOCKER_HUB_USERNAME }}
                   password: ${{ secrets.DOCKER_HUB_TOKEN }}
 
-            # - name: Set Yarn Version
-            #   run: yarn version --new-version ${{ env.THIS_VERSION }}
+            - name: Build Container and export to Docker
```

Comment:
Not sure I understand if the current build ensures the go app (ci.yaml -> build) and react app (release.yml -> release) are both successfully built with the latest code. Meaning, we might need a "needs" directive somewhere in the CI configs to make sure a successful go build precedes the react/docker builds.

Reply:
If go build fails (non-zero exit code), job would exit with failure if that's what you meant @bhamail?

Reply:
In GHActions, each "job" is "independent" of other jobs. So while the go build will fail with non-zero exit code, that failure would not prevent a separate run of the "release" job from succeeding. It seems like there is a possibility of deploying with a "stale" go app, but that possibility is unlikely.

I'm alluding to possible need to create some "dependency" between jobs where the "react app" job should possibly  "depend" on a successful build of the "go app", thereby ensuring the deployed docker image is always "up to date".
It's an edge case that may not be worth worrying about, but feels like it does exist in the current CI config.